src/xyPlot.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from abaqus.XY.AreaStyle import AreaStyle
+from abaqus.XY.LineStyle import LineStyle
+from abaqus.XY.QuantityType import QuantityType
+from abaqus.XY.SymbolStyle import SymbolStyle
+from abaqus.XY.TextStyle import TextStyle
+from abaqus.XY.XYData import XYData
+from abaqus.XY.XYSession import XYSession
+
+# TODO: signature for these functions, see if it is really necessary
+
+def XYDataFromFile() -> XYData:
+    ...
+def XYDataFromFreeBody() -> List[XYData]:
+    ...
+def XYDataFromHistory() -> XYData:
+    ...
+def XYDataFromPath() -> XYData:
+    ...
+def XYDataFromShellThickness() -> List[XYData]:
+    ...
+def xyDataListFromField() -> List[XYData]:
+    ...
```

Comment:
I think we'd better add the signatures and docstrings

Reply:
Or move this into somewhere inside the abaqus package, and import it here

Reply:
Ok, I'll add the signatures and docstrings.
They would be the same as in [XYData module](https://github.com/haiiliin/abqpy/blob/3717cad286a7d64c3aff3cb06e3c31366f115382/src/abaqus/XY/XYData.py#L192), is that correct?

Reply:
Yes, I think so

Reply:
Done on [c29533c](https://github.com/haiiliin/abqpy/pull/1068/commits/c29533c80cd3671d7f3a92df9ead3903aafd7107)